quil-rs/src/program/graph.rs

```diff
@@ -461,7 +463,7 @@ impl InstructionBlock {
 
     /// Return a particular-indexed instruction (if present).
     pub fn get_instruction(&self, node_id: usize) -> Option<&Instruction> {
-        self.instructions.get(node_id)
+        self.instructions.get(node_id).copied()
```

Comment:
Another case of `&&Instruction`. Since references are `Copy`, this is effectively a single dereference.